src/devtools/client/debugger/src/components/Editor/index.js

```diff
@@ -333,16 +332,6 @@ class Editor extends PureComponent {
     }
   }
 
-  setSize(nextProps, editor) {
-    if (!editor) {
-      return;
-    }
-
-    if (nextProps.startPanelSize !== this.props.startPanelSize) {
-      editor.codeMirror.setSize();
```

Comment:
The [docs on this](https://codemirror.net/doc/manual.html#setSize) indicate that passing null effectively means "don't change anything", so this should be a noop. I've checked it in the browser and indeed I cannot tell any difference between leaving this code in and removing it. Lmk if I'm missing something here!

Reply:
That looks to be right 👍